sdk/monitor/monitor-opentelemetry-exporter/src/config.ts

```diff
@@ -11,30 +11,37 @@ import {
 const DEFAULT_BATCH_SEND_RETRY_INTERVAL_MS = 60_000;
 const DEFAULT_MAX_CONSECUTIVE_FAILURES_BEFORE_WARNING = 10;
 
+/**
+ * Provides configuration options for AzureMonitorTraceExporter.
+ */
 export interface AzureExporterConfig {
-  // Setup String
+  /**
+   * Azure Connection String.
```

Comment:
```suggestion
   * Azure Monitor Connection String.
```
(or maybe the product name is different than that)

It might also be useful to include a little bit of the connection string format so people don't assume they can paste a bare instrumentation key in there.